### PR TITLE
Implement compose framework for Dropdown

### PR DESCRIPTION
**Pull request by rurikoaraki**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Skeletal compose framework for dropdown. Getting all the scaffolding up to make implementing the actual changes easier.

### Verification

This component doesn't do anything. Build succeeds

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: packages/experimental/Dropdown/package.json [hunk 1]
```diff
@@ -25,7 +25,11 @@
     "url": "https://github.com/microsoft/fluentui-react-native.git",
     "directory": "packages/experimental/dropdown"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@fluentui-react-native/adapters": ">=0.9.0 <1.0.0",
```
Comment (rurikoaraki) [inline packages/experimental/Dropdown/package.json:29]: Reminder: check that these haven't updated before merging

File: change/@fluentui-react-native-experimental-switch-c30f2890-78de-40b6-8126-6eeb6d700039.json [hunk 2]
```diff
@@ -1,7 +0,0 @@
-{
-  "type": "minor",
-  "comment": "Added Switch control for win32",
```
Comment (Saadnajmi) [inline change/@fluentui-react-native-experimental-switch-c30f2890-78de-40b6-8126-6eeb6d700039.json:3]: Intentional?

**Review by FalseLobster — APPROVED** [review]

File: packages/experimental/Dropdown/src/Dropdown/Dropdown.styling.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { UseStylingOptions } from '@fluentui-react-native/framework';
+import { dropdownName, DropdownProps, DropdownSlotProps, DropdownTokens } from './Dropdown.types';
+
+export const stylingSettings: UseStylingOptions<DropdownProps, DropdownSlotProps, DropdownTokens> = {
+  tokens: [dropdownName],
```
Comment (Saadnajmi) [inline packages/experimental/Dropdown/src/Dropdown/Dropdown.styling.ts:5]: The name is a token?

**Review by Saadnajmi — APPROVED** [review]

File: change/@fluentui-react-native-experimental-switch-c30f2890-78de-40b6-8126-6eeb6d700039.json (hunk 2 above)
Comment (rurikoaraki) [reply change/@fluentui-react-native-experimental-switch-c30f2890-78de-40b6-8126-6eeb6d700039.json:3]: Yes, beachbball has been complaining about it so I deleted it. Probably shouldn't be part of this PR, but oh well.

File: packages/experimental/Dropdown/src/Dropdown/Dropdown.styling.ts (hunk 3 above)
Comment (rurikoaraki) [reply packages/experimental/Dropdown/src/Dropdown/Dropdown.styling.ts:5]: Yeah, it generally has been if you look at other components. Why? not sure, maybe we shouldn't be doing it? or maybe it helps with some sort of framework lookup stuff.

**Pull request merged**